plugin-gradle/CHANGES.md

```diff
@@ -3,6 +3,9 @@
 We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (starting after version `3.27.0`).
 
 ## [Unreleased]
+### Added
+* **POTENTIALLY BREAKING** Support Ktlint 1.0.0 ([#1808](https://github.com/diffplug/spotless/pull/1808)).
+  The default code style had been changed to `ktlint_official`, if you are upgrading from the old versions, may need to override the style in `editorConfigOverride`.
```

Comment:
Two questions:

1) Has the default style changed for all ktlint versions, e.g. `0.49.0`? Or only for `1.0.0`?
2) How hard is it to switch the default style in Spotless to be the same as it was, apparently `intellij_idea`?

IMO, the user we should care the most about is the user like this:

```gradle
spotless {
  kotlin {
    ktlint()
  }
}
```

This is the user who cares the least about what their format is, and is hoping to delegate that to us completely. We have to break them a little bit as we upgrade versions which fix bugs, but we should try to make their experience as seamless as possible.

I don't care as much if this user has to add flags to keep their code formatting the same with a new version of Spotless

```gradle
spotless {
  kotlin {
      .editorConfigOverride(
        mapOf(
          "indent_size" to 2,
        )
      )
```

Reply:
1. Only 1.0.0
2. I tried to modify `com.diffplug.spotless.glue.ktlint.compat.KtLintCompat1Dot0Dot0Adapter#DEFAULT_EDITOR_CONFIG_PROPERTIES`, but 7de7991d86f867ed7c528d915eec493c2699be25 for now.